doc/file_io.rst

```diff
@@ -54,6 +54,7 @@ File I/O
    what
    io.read_info
    io.show_fiff
+   io.pick.get_channel_type_constants
```

Comment:
This was documented as a public attribute in `whats_new.rst` so I guess we live with it :shrug:

Reply:
we can always deprecate. hard to believe it would be very useful to end-users.

Reply:
Okay, done -- moved to `mne.io.get_channel_type_constants` (it's implicitly deprecated via the magic `__getattr__`)